deepseg.cc

```diff
@@ -164,23 +293,23 @@ int main(int argc, char* argv[]) {
 				showUsage = true;
 			}
 		} else if (strncmp(argv[arg], "-w", 2)==0) {
-			if (hasArgument && sscanf(argv[++arg], "%d", &width)) {
+			if (hasArgument && sscanf(argv[++arg], "%ld", &width)) {
```

Comment:
With `width`, `height`, and `threads` being `size_t` it probably should be `%zu` for the format code.
Check the `-Wformat=2` compiler option from `gcc` and `clang`

Reply:
done